### DISCUSSION TITLE
Compiling problems - missing dac.h

### DISCUSSION THREAD

**Discussion by svake** [davetcc/tcMenu#325] [General]
Hello,

when I try to compile the TC menu I generated I get this errror:
\Arduino\libraries\IoAbstraction\src\esp32\ESP32AnalogDevice.cpp:14:10: fatal error: driver/dac.h: No such file or directory
 #include <driver/dac.h>
I had the latest versions of everything installed, for IoAbstraction I tried a downgrade to 3.02 without positiv result. 
I checked the folder and driver, both are not existing. I even downloaded a dac.h from expressiv, but that was also not helpful. I may have made a mistake somewhere because searching this topic for hours did not yield results.
I use the XIAO ESP32C3 for it's small size and Arduino 2.04

All the best,
Svake

**Comment by davetcc** [comment]
Apologies for some reason I thought this was an issue that why I used the blanket close response

**Reply by davetcc** [reply]
I’ll look into this and get back, but only the s3 is currently supported.
